ChangeLog

```diff
@@ -7,6 +7,10 @@ Bugfix
      invalidated keys of a lifetime of less than a 1s. Fixes #1968.
    * Fix failure in hmac_drbg in the benchmark sample application, when
      MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
+   * Zeroize memory used for buffering or reassembling handshake messages
```

Comment:
I am not certain if these are actually bug fixes. Wouldn't "changes" be a more appropriate category?

Reply:
I think a lack of zeroization, or an unreliable attempt to zeroize through `memset()`, is a bug (and sometimes even a security flaw).